lightkurve/lightcurve.py

```diff
@@ -618,7 +618,8 @@ def _create_plot(self, method='plot', ax=None, normalize=True,
                 fig, ax = plt.subplots(1)
             if method == 'scatter':
                 sc = ax.scatter(self.time, flux, **kwargs)
-                if show_colorbar and ('c' in kwargs) and hasattr(kwargs['c'], '__iter__'):
+                if show_colorbar and ('c' in kwargs) and \
+                  (not isinstance(kwargs['c'], str)) and hasattr(kwargs['c'], '__iter__'):
```

Comment:
Let's add a comment to explain that we need this condition to avoid showing a colorbar when a fixed color is set using a string.  The purpose of this if statement will not be obvious otherwise.

Reply:
👍